gdx/src/com/badlogic/gdx/math/MathUtils.java

```diff
@@ -59,22 +59,26 @@
 		}
 	}
 
-	/** Returns the sine in radians from a lookup table. */
+	/** Returns the sine in radians from a lookup table. For optimal precision, use radians that are not drastically more positive

```

Comment:
Is this the same as: "For optimal precision use radians in the range [-PI2, PI2]."?
If so in my opinion this would be more understandable.

Reply:
Phrasing is going to be tricky here. I can try to figure out where it starts losing precision, but it's probably somewhere greater than `PI2` or less than `-PI2`. The lookup table has 18 bits of precision I think, with 65536 entries corresponding to one quadrant, but it could be lower. That might be just 16 bits of precision, depending on how you measure it. A float has 23 mantissa bits, and those represent larger and larger units as the exponent increases, so you'd only run out of mantissa outside `(-32,32)` or `(-128,128)`, and I'll need to run some tests to make sure. Even then, the difference is small around that limit, but if you don't somehow limit how high or low the `radians` argument goes, it is very easy to exceed the precise range.

But yeah, I like your comment text better except for the quirk where not every school system teaches that syntax for a closed interval (it might be like the local difference between `1,000,000` and `1.000.000`, and I don't know specifics other than sometimes people don't understand when I use that syntax). Maybe `"For optimal precision, use radians between -PI2 and PI2, both inclusive."`